### ISSUE TITLE
PicoTorrent 0.8.0 can't start

### ISSUE THREAD

**Issue by smaragdus** [picotorrent/picotorrent#214]
PicoTorrent 0.7.0 worked for me but PicoTorrent 0.8.0 doesn't- it fails to start (portable edition). 

Screen-shots:
http://postimg.org/image/utwkr8csp/
http://postimg.org/image/ohm0v4z49/

What is the difference between "PicoTorrent-0.8.0-x64.zip" and "PicoTorrent-0.8.0-x64.portable.zip"?

**Comment by zp** [comment]
`PicoTorrent-0.8.0-x64.zip` is the "non installer" zip, which only contains the base PicoTorrent files.

`PicoTorrent-0.8.0-x64.portable.zip` is the "true portable" zip that contains PicoTorrent + all of the required runtime dll files

**Comment by vktr** [comment]
@smaragdus If 0.7 worked for you, then I guess you have the C++ runtime installed? If so, the regular 0.8 package should work fine.

**Comment by smaragdus** [comment]
@zp
Thanks for the clarification. I re-tested all non-installer versions of PicoTorrent 0.8.0 and the only one which ran fine was "PicoTorrent-0.9.0-x64.zip", all the other versions spew the error messages I posted above.

@vktr
Yes, PicoTorrent 0.7.0 was working fine, but PicoTorrent 0.8.0 wasn't, that was why I submitted the issue. And yes, I have Visual C++ Redistributable 2015 14.0.23506.0 installed (both x32 and x64 versions).

I have just tested PicoTorrent 0.9.0 and there are no more error messages so I suppose PicoTorrent 0.8.0 was a faulty release. "PicoTorrentCore.dll" isn't bundled in version 0.9.0, am I supposed to delete it from PicoTorrent program folder (remnant from version 0.7.0)?
